dashboard/app/views/levels/_admin.html.haml

```diff
@@ -73,6 +73,14 @@
             = link_to build_script_level_path(script_level), build_script_level_path(script_level)
       %br/
       .project_admin
+      %br/
+      - storage_app_id = storage_decrypt_channel_id(request.url.split("/").last).first
+      - featured = FeaturedProject.exists?(:storage_app_id => storage_app_id)
+      - featured_project_id = featured ? FeaturedProject.find_by_storage_app_id(storage_app_id).id : nil
+      #unfeature_project{'style' => ('display: none;' unless featured)}
+      #feature_project{'style' => ('display: none;' if featured)}
+      = content_tag 'div', '', class: 'featured_project_id', data: {id: featured_project_id}
```

Comment:
my previous comment describes how to avoid needing to know if a project is `featured` in ruby/haml. it seems like you shouldn't need the `featured_project_id` here either, if your API is going to accept the channel id instead.